src/main/java/de/samply/common/ldmclient/cql/LdmClientCql.java

```diff
@@ -158,6 +173,81 @@ private QueryResultStatistic createQueryResultStatistic(MeasureReport measureRep
     return queryResultStatistic;
   }
 
+  /**
+   * Create a sublist for an entity.
+   *
+   * @param location the measure URI
+   * @return location url for the sublist
+   * @throws LdmClientException exception which can be thrown while posting the query to the ldm
+   */
+  public String createSubList(URI location) throws LdmClientException {
+    JsonObject parameter = loadParameterStub();
+    location = location.resolve("/$evaluate-measure");
+    HttpPost httpPost = new HttpPost(location);
+    httpPost.setHeader(HttpHeaders.CONTENT_TYPE, "application/fhir+json");
+    HttpEntity entity = new StringEntity(parameter.toString(), Consts.UTF_8);
+    httpPost.setEntity(entity);
+    try (CloseableHttpResponse response = getHttpClient().execute(httpPost)) {
+      int statusCode = response.getStatusLine().getStatusCode();
+      if (statusCode != HttpStatus.SC_CREATED) {
+        logger.error(String.format("Measure report not created. Status code: %d, Response: %s",
+            statusCode, EntityUtils.toString(response.getEntity(), Consts.UTF_8)));
+        throw new LdmClientException("Request not created. Received status code " + statusCode);
+      }
+      Header locationHeader = response.getFirstHeader("Location");
+      if (locationHeader == null) {
+        throw new LdmClientException("Location header is missing");
+      }
+      String measureReportUri = resourceLocation(locationHeader.getValue());
+      if (measureReportUri.equals("")) {
+        throw new LdmClientException("Location header is empty");
+      }
+      return getFirstSubjectListUri(measureReportUri);
+    } catch (IOException e) {
+      throw new LdmClientException(e);
+    }
+  }
+
+  private String getFirstSubjectListUri(String measureReportUri) throws LdmClientException {
+    HttpGet httpGet = new HttpGet(measureReportUri);
+    httpGet.setHeader(HttpHeaders.ACCEPT, "application/fhir+json");
+    try (CloseableHttpResponse response = getHttpClient().execute(httpGet)) {
+      int statusCode = response.getStatusLine().getStatusCode();
+      if (statusCode == HttpStatus.SC_OK) {
+        HttpEntity entity = response.getEntity();
+        String entityOutput = EntityUtils.toString(entity, Consts.UTF_8);
+        MeasureReport measureReport = (MeasureReport) FHIR_CONTEXT.newJsonParser()
+            .parseResource(entityOutput);
+        return LdmClientUtil.addTrailingSlash(getLdmBaseUrl()) + measureReport.getGroupFirstRep()
+            .getPopulationFirstRep()
+            .getSubjectResults()
+            .getReference();
+      } else {
+        throw new LdmClientException(
+            "Unexpected response code '" + statusCode
+                + "' while getting sublist from measure with URL '"
```

Comment:
This should be called subject list, because sublist sounds like the part of a bigger list.